spec/github_helper_spec.rb

```diff
@@ -19,4 +19,27 @@
       end
     end
   end
+
+  describe 'github_token' do
+    after do
+      ENV['GHHELPER_ACCESS'] = nil
+      ENV['GITHUB_TOKEN'] = nil
+    end
+
+    it 'can use `GHHELPER_ACCESS`' do
+      ENV['GHHELPER_ACCESS'] = 'GHHELPER_ACCESS'
+      expect(described_class.github_token).to eq('GHHELPER_ACCESS')
+    end
+
+    it 'can use `GITHUB_TOKEN`' do
+      ENV['GITHUB_TOKEN'] = 'GITHUB_TOKEN'
+      expect(described_class.github_token).to eq('GITHUB_TOKEN')
+    end
+
+    it 'prioritizes GHHELPER_ACCESS` over `GITHUB_TOKEN` if both are present' do
```

Comment:
I wonder if we should not instead consider that `GHHELPER_ACCESS` is deprecated, meaning that while we'd still keeping support for it for a while like you did, `GITHUB_TOKEN` should be considered the "new way" and thus the one being prioritized instead?

Reply:
Ah didn't think about the breaking change behavior, very good point!